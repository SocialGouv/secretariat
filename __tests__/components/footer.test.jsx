import { render } from "@testing-library/react"

import Footer from "../../src/components/footer"
import { it, expect } from "vitest"

it("renders footer", () => {
  const { container } = render(<Footer />)
  expect(container).toMatchSnapshot()
})
