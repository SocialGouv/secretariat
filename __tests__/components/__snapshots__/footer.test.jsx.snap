// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders footer 1`] = `
<div>
  <footer>
    <div
      class="footer-content"
    >
      <div
        class="logo-info"
      >
        <div
          class="logo big"
        >
          République
          <br />
          Française
        </div>
        <div
          class="info"
        >
          <p>
            <strong>
              Secretariat
            </strong>
             est une application permettant de gérer les accès des utilisateurs aux services de la Fabrique Numérique.
          </p>
          <p
            class="flex items-center"
          >
            <i
              class="ri-github-fill ri-xl relative top-px mr-1"
            />
            <a
              href="https://github.com/SocialGouv/secretariat/tree/v0.0.42"
              rel="noreferrer"
              target="_blank"
            >
              version 
              0.0.42
            </a>
          </p>
        </div>
      </div>
      <div
        class="links"
      >
        <a
          href="https://www.fabrique.social.gouv.fr/"
          rel="noreferrer"
          target="_blank"
        >
          La Fabrique Numérique
        </a>
        <a
          href="/legal"
        >
          Mentions légales
        </a>
        <a
          href="/confidentialite"
        >
          Politique de confidentialité
        </a>
        <button
          aria-label="theme toggle"
        >
          <i
            class="theme-toggle-button ri-sun-line"
          />
        </button>
      </div>
    </div>
  </footer>
</div>
`;
